src/libData/AccountData/Account.cpp

```diff
@@ -255,6 +545,23 @@ vector<string> Account::GetStorage(string _k) const
     return {rlp[1].toString(), rlp[2].toString(), rlp[3].toString()};
 }
 
+string Account::GetRawStorage(const h256& k_hash) const
+{
+    if (!isContract())
+        return "";
+    return m_storage.at(k_hash);
+}
+
+vector<h256> Account::GetStorageKeyHashes() const
```

Comment:
Do you need mutex to protect m_storage?

Reply:
Not essential as no operations on AccountStore will happen in parallel for now